### PR TITLE
Added XPT2046 touch screen digitizer for ILI9341 display

### PR DESCRIPTION
**Pull request by nonix**
## Description:

**Related issue (if applicable):** fixes #<Tasmota issue number goes here>

## Checklist:
  - [ x] The pull request is done against the latest development branch
  - [ x] Only relevant files were touched
  - [ x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [ x] The code change is tested and works on Tasmota core ESP8266 V.2.7.4.9
  - [ ] The code change is tested and works on Tasmota core ESP32 V.1.0.5-rc6
  - [x ] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

_NOTE: The code change must pass CI tests. **Your PR cannot be merged unless tests pass**_

### CODE REVIEW

**Comment by nonix** [comment]
This change extends Tasmota functionality with LoLin TFT 2.4" touch screen display. The LoLin TFT 2.4" is Wemos D1 R2 mini add-on hat with TFT driver ILI9341 and restive touch screen driver XPT2046.

Note: this is my first contribution, please accept my apology if not all code is as expected.

**Comment by ascillato** [comment]
What is the use for the file xpt.patch?

**Comment by nonix** [comment]
That's my mistake, the file should not have been committed. I can try to remove ... Sorry.

**Comment by nonix** [comment]
I have removed the unnecessary xpt.patch (my temp) file.

File: tasmota/tasmota_template.h [hunk 1]
```diff
@@ -152,6 +152,9 @@ enum UserSelectablePins {
   GPIO_TM1637CLK, GPIO_TM1637DIO,      // TM1637 interface
   GPIO_PROJECTOR_CTRL_TX, GPIO_PROJECTOR_CTRL_RX,  // LCD/DLP Projector Serial Control
   GPIO_SSD1351_DC,
+#ifdef USE_XPT2046
+  GPIO_XPT2046_CS,                     // XPT2046 SPI Chip Select
+#endif
   GPIO_SENSOR_END };
```
Comment (arendst) [inline tasmota/tasmota_template.h:158]: Do not use ``#ifdef USE_XPT2046`` here as it needs to be compiled in always to keep the list complete.

So pls remove these (only here).

Comment (nonix) [reply tasmota/tasmota_template.h:156]: Done, ifdefs removed and committed

**Pull request merged**